src/ekore/anomalous_dimensions/unpolarized/time_like/as1.py

```diff
@@ -0,0 +1,156 @@
+"""The unpolarized LO Altarelli-Parisi splitting kernels."""
```

Comment:
```suggestion
"""The unpolarized |LO| Altarelli-Parisi splitting kernels."""
```
and similar

Reply:
done

Reply:
please also modify the docstrings of the fncs

Reply:
also done